### PR TITLE
SAK-32537 Add calendar link to user drop-down menu.

### PR DESCRIPTION
**Pull request by RebeccaOxfordUni**
Note that the link is not available when logged in as the admin user as this user does not have the calendar tool.

postadditionoflink jpp

### CODE REVIEW

**Comment by jonespm** [comment]
I don't know if this link adds much value to the drop down. It seems like having a dropdown on the "Home" tab would be better since someone could argue they'd want to get to "announcements" or "resources" in their home just as much as calendar.

**Comment by fostersdesign** [comment]
We discussed this on the Sakai Core team call today. The team recommended that we have the UX group discuss the content of this menu. 

For example, the "Create New Site" link should be removed because it isn't account-related and the link also exists in the Sites drawer. 

Matt's comment above was also discussed: if Calendar is added, then people/institutions may want to add Resources, Account, Announcements, and any other tools that they have in Home to this menu. At that point, we're re-creating the tool down-arrow menu that most sites have across the top; maybe we just need to re-add this tool menu back to home, which will give users quick access to these tools.

I will post this PR on the UX list to see what the group says.

**Comment by lardo-de-arnaud** [comment]
Aside of whether this idea is acceptable to the community, would somebody be able to look at the code for correctness? just make comments - no need to actually pass/accept the PR.

**Comment by buckett** [comment]
Code wise I think this is fine.

**Comment by ottenhoff** [comment]
Any feedback from UX group?

**Comment by lardo-de-arnaud** [comment]
Shawn was very apologetic, he forgot to ask.

I created a parent Jira at the request of Earle. This issue is linked to it.

adam

File: portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/site/MoreSiteViewImpl.java [hunk 1]
```diff
@@ -149,6 +151,8 @@ public Object getRenderContextObject()
                             if ( profileToolId.equals(placement.getToolId()) ) {
                                 profileToolUrl = Web.returnUrl(request, "/site/" + Web.escapeUrl(siteHelper.getSiteEffectiveId(s)) + "/page/" + Web.escapeUrl(p.getId()));
                                 mrphs_profileToolUrl = Web.returnUrl(request, "/site/" + Web.escapeUrl(siteHelper.getSiteEffectiveId(s)) + "/tool-reset/" + Web.escapeUrl(placement.getId()));
+							} else if ( calendarToolId.equals(placement.getToolId()) ) {
```
Comment (buckett) [inline portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/site/MoreSiteViewImpl.java]: Indentation looks off here.

**Comment by buckett** [comment]
Having the user dropdown be the same entries as My Workspace is one option, but if that's a request could we open a new issue for it. If people really don't want this could we just change the default to be `none` and put this in?

**Comment by fostersdesign** [comment]
I posted a message to the Sakai-UX mailing list on May 23rd and Adam M. was the only person who responded to my message. No additional UX group input has been discussed, as far as I'm aware.

**Comment by lardo-de-arnaud** [comment]
I asked the dev / user community and everybody is happy with this going in. The only issue is, I believe, of of indentation. Once that's fixed, could somebody merge?

**Comment by RebeccaOxfordUni** [comment]
Indentation issue mentioned by Matthew fixed.

**Pull request merged**